src/styles/components/banner/styles.less

```diff
@@ -0,0 +1,7 @@
+& when (@banner-enabled) {
+
+  .banner-container {
```

Comment:
Instead of this, wondering why we couldn't wrap the Banner JSX in a `<div className="pod">`? We could potentially avoid all custom CSS.